deployment/terraform/coordinator.go

```diff
@@ -113,9 +115,34 @@ func (t *Terraform) StartCoordinator() error {
 	}
 
 	mlog.Info("Starting the coordinator")
-	cmd := "sudo service ltcoordinator start"
-	if out, err := sshc.RunCommand(cmd); err != nil {
-		return fmt.Errorf("error running ssh command: output: %q, error: %w", out, err)
+	data, err = json.Marshal(struct {
+		CoordinatorConfig coordinator.Config
+		LoadTestConfig    loadtest.Config
+	}{
+		*coordinatorConfig,
+		*agentConfig,
+	})
+	if err != nil {
+		return err
+	}
+	// TODO: create a wrapper to simplify coordinator's HTTP API usage.
+	id := fmt.Sprintf("%s-coordinator-%d", t.config.ClusterName, 0)
```

Comment:
Doesn't this restrict one user to run only one coordinator? Or you are saying the differentation comes from the cluster name? In which case, we should just remove the "-0" part since that doesn't really mean anything.

Reply:
The idea was to address that in [MM-26387](https://mattermost.atlassian.net/browse/MM-26387). Here really trying to get things working as they are now (before the two PRs merged) and just preparing the code for what's next.